load-responsively.js

```diff
@@ -32,9 +32,14 @@
       $container.css({ opacity: 0, transition: 'opacity 1s ease-in-out' });
       $this.css({position: 'absolute', top: 0, left: 0, width: '100%'});
 
-      $image.on('load', function() {
+      if ($image.width !== undefined && $image.width !== 0) {
```

Comment:
i guess the difference would be that if it is done that way, it will attach unnecessary event handlers. The way i'm doing it the event handler only gets attached if it hasn't loaded already.